sparse/_coo/core.py

```diff
@@ -1451,6 +1451,60 @@ def asformat(self, format, compressed_axes=None):
 
         raise NotImplementedError("The given format is not supported.")
 
+    def pad(self, pad_width, mode="constant", **kwargs):
+        """
+        Performs the equivalent of :obj:`numpy.pad` for :obj:`COO`. Note that
+        this function returns a new array instead of a view.
+
+        Parameters
+        ----------
+        pad_width : {Sequence, Array_like, Int}
```

Comment:
```suggestion
        pad_width : int
```

That's the only one we support.

Reply:
Sir pad_width can be arbitrary just like numpy

Reply:
Ah, okay. Then can you make it `{np.ndarray, int}`

Reply:
Sir we can also send a sequence of tuples of lists also. I have checked it it's working.

Reply:
```suggestion
        pad_width : {sequence, array_like, int}
```